### DISCUSSION TITLE
using several GPUs on a single workstation

### DISCUSSION THREAD

**Discussion by errinaceus** [KosinskiLab/AlphaPulldown#563] [Q&A]
Is there a way to use two GPUs on a single workstation, not on a cluster?
export CUDA_VISIBLE_DEVICES doesn't work, neither does AlphaFold's flag --gpu_devices

**Comment by kashyapchhatbar** [comment]
Here's what we've adapted in our workflow. Create a Docker image with AlphaPulldown configured and fire up one container with `--gpu device=0` and another with `--gpu device=1`. Split your `custom_mode.txt` in two parts and let the containers run parallel.

**Comment by jkosinski** [comment]
I think it is a reasonable feature request and a good idea, and maybe could be moved back to Issues?
